Hari orgid

dev4pulse opened the pull request.

Sriharip777 committed:
product, inventory and inventory log features created

Sriharip777 committed:
crud operations updated

dhanunjay20 committed:
Merge branch 'hari_orgid' into hari_products

dhanunjay20 committed:
Merge pull request #3 from dhanunjay20/hari_products

Hari products

Sriharip777 committed:
Enhance InventoryLogServiceImpl and controllers with logging, try-catch blocks, and detailed comments

jangoamranjith commented on pom.xml:
```diff
@@ -80,7 +75,9 @@
 						</path>
 					</annotationProcessorPaths>
-				</configuration>
+                    <source>25</source>
```
Please add top as variable in the property section

jangoamranjith commented on src/main/java/com/pulse/fineflux/controller/InventoryController.java:
```diff
@@ -0,0 +1,98 @@
+
+            InventoryResponseDTO response = inventoryService.createInventory(dto);
+
+            log.debug("Inventory created successfully inventoryId={}", response.getInventoryId());
```
change this info log

jangoamranjith commented on src/main/java/com/pulse/fineflux/controller/InventoryController.java:
```diff
@@ -0,0 +1,98 @@
+
+            List<InventoryResponseDTO> updatedList = inventoryService.updateInventory(orgId, productId, dto);
+
+            log.debug("Inventory updated successfully for productId={}", productId);
```
change this to info

jangoamranjith commented on src/main/java/com/pulse/fineflux/controller/InventoryController.java:
```diff
@@ -0,0 +1,98 @@
+
+            List<InventoryResponseDTO> inventories = inventoryService.getAllInventories(orgId);
+
+            log.debug("Fetched {} inventories for orgId={}", inventories.size(), orgId);
```
change this to info same

jangoamranjith reviewed (approved):
Minor: log changes

Sriharip777 committed:
- Implemented new modules for Collection, Sales, and GunInfo
- Added corresponding entities, repositories, services, and controllers
- Integrated endpoints for CRUD operations and organization-based filtering
- Tested all modules through Postman with debug logging enabled

Sriharip777 committed:
create profit_loss module

Sriharip777 committed:
feat: integrated profit_loss module with sales and inventory auto-update
